JavaScript/JavaScriptSDK/ajax/ajax.ts

```diff
@@ -132,7 +134,11 @@ module Microsoft.ApplicationInsights {
         }
 
         private sendHandler(xhr: XMLHttpRequestInstrumented, content) {
-            xhr.ajaxData.requestSentTime = dateTime.Now();            
+            xhr.ajaxData.requestSentTime = dateTime.Now();
+            if (!this.appInsights.config.disableCorrelationHeaders && (UrlHelper.parseUrl(xhr.ajaxData.getAbsoluteUrl()).host == this.currentWindowHost)) {
```

Comment:
why have currentWindowHost restriction? Host name may be different even for the same component and going forward we'll look into supporting it across components.